zfs.go

```diff
@@ -20,16 +20,14 @@ package zfs
 
 import (
 	"context"
-	"path/filepath"
-
+	_ "crypto/sha256"
```

Comment:
Why was this added?

Reply:
I was getting the following error before adding this import.

```
panic: sha256 not available (make sure it is imported) [recovered]
	panic: sha256 not available (make sure it is imported)
```

Reply:
Since this package does not directly use it, it is not the normally the place you would import it. Where was it importing from that ran into this error? You can import it from `_test.go` file if that is necessary.

Reply:
@sedflix ^^ could you have a look?

Reply:
@dmcgowan i shifted it to zfs_test.go. lmk if any other changes are required are here.